modules/nw-ovn-ipsec-north-south-enable.adoc

```diff
@@ -18,16 +18,82 @@ After you apply the machine config, the Machine Config Operator reboots affected
-* You have installed the `butane` utility.
 * You have an existing PKCS#12 certificate for the IPsec endpoint and a CA cert in PEM format.
 
 .Procedure
 
 As a cluster administrator, you can enable IPsec support for external IPsec endpoints.
 
-. Create an IPsec configuration file named `ipsec-endpoint-config.conf`. The configuration is consumed in the next step. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/configuring-a-vpn-with-ipsec_securing-networks#configuring-a-vpn-with-ipsec_securing-networks[Libreswan as an IPsec VPN implementation].
+. To enable IPsec encryption, enter the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge \
+-p '{
+  "spec":{
+    "defaultNetwork":{
+      "ovnKubernetesConfig":{
+        "ipsecConfig":{
+          "mode":<mode>
+        }}}}}'
+----
++
+where:
++
+--
+`mode`:: The IPsec behavior. Specify `External` to encrypt only traffic to external hosts or specify `Full` to encrypt pod to pod traffic and traffic to external hosts. By default, IPsec is disabled.
+--
+
+. Create an IPsec configuration with an NMState Operator node network configuration policy. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/configuring-a-vpn-with-ipsec_securing-networks#configuring-a-vpn-with-ipsec_securing-networks[Libreswan as an IPsec VPN implementation].
+
+.. To identify the IP address of the cluster node that is the IPsec endpoint, enter the following command:
++
+----
+$ oc get nodes
+----
+
+.. Create a file named `ipsec-config.yaml` that contains the node network configuration policy for the NMState Operator:
++
+.Example NMState configuration
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: ipsec-config
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: "<hostname>" <1>
+  desiredState:
+    interfaces:
+    - name: <interface_name> <2>
+      type: ipsec
+      libreswan:
+        left: <left_ip_address> <3>
+        leftid: '%fromcert'
+        leftrsasigkey: '%cert'
+        leftcert: left_server
+        right: <right_ip_address> <4>
+        rightid: '%fromcert'
+        rightrsasigkey: '%cert'
+        ikev2: insist
+        type: <transport_type>
+----
+<1> Specifies the host name to apply the policy to. This host serves as the left side host in the IPsec configuration.
+<2> Specifies the name of the interface to create on the host.
+<3> option
+<4> option
+
+.. To configure the IPsec interface, enter the following command:
++
+[source,terminal]
+----
+$ oc create -f ipsec-config.yaml
+----
 
 . Provide the following certificate files to add to the Network Security Services (NSS) database on each host. These files are imported as part of the Butane configuration in subsequent steps.
 +
```

Comment:
the mc should not include the os extension and the systemd unit anymore.
it is strictly for the certificates

Reply:
and we might wanna note that even that is just an example
customer for example, can use a mc per mcp (if he created other mcps)